### PR TITLE
Update outdated guidance for adding on-premises MDM application

### PR DESCRIPTION
**Pull request by andreiztm**
Customers were confused by previous guidance, as it was outdated/incorrect.

### CODE REVIEW

File: windows/client-management/mdm/azure-active-directory-integration-with-mdm.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ For more information about how to register a sample application with Azure AD, s
 
 An on-premises MDM application is inherently different that a cloud MDM. It is a single-tenant application that is present uniquely within the tenant of the customer. Therefore, customers must add the application directly within their own tenant. Additionally, each instance of an on-premises MDM application must be registered separately and has a separate key for authentication with Azure AD.
 
-The customer experience for adding an on-premises MDM to their tenant is similar to that as the cloud-based MDM. There is an entry in the Azure AD app gallery to add an on-premises MDN to the tenant and administrators can configure the required URLs for enrollment and Terms of Use.
+To add an on-premises MDM application to the tenant there is an entry under the Azure AD service, specifically under Mobility (MDM and MAM) -> Add application. Administrators can configure the required URLs for enrollment and Terms of Use.
```
Comment (illfated) [inline windows/client-management/mdm/azure-active-directory-integration-with-mdm.md:139]: Readability comma (minor edit)
```suggestion
To add an on-premises MDM application to the tenant, there is an entry under the Azure AD service, specifically under Mobility (MDM and MAM) -> Add application. Administrators can configure the required URLs for enrollment and Terms of Use.
```

**Comment by andreiztm** [comment]
Thank you @illfated :D.

**Comment by andreiztm** [comment]
Just as FYI, we also have a CI item open (116208) for an entire review of this page, but this one paragraph couldn't wait, as it what caused some customer grief. @JarrettRenshaw is aware of the CI item.

**Comment by illfated** [comment]
Very well, thanks for the heads up. Based on your comment, I will leave it to you and your colleagues to decide how fast you want to merge this content. The change in itself is good to go, as far as I am concerned.

**Comment by illfated** [comment]
#sign-off

**Comment by andreiztm** [comment]
Perfect, thanks again for this help. This one should be merged as soon as we can.

**Pull request merged**